package.json

```diff
@@ -87,5 +87,9 @@
       "./shims/yarnpkg",
       "./shims/yarnpkg.ps1"
     ]
+  },
+  "dependencies": {
+    "babel-plugin-dynamic-import-node": "^2.3.3",
+    "v8-compile-cache": "^2.3.0"
```

Comment:
These should probably be in `devDependencies`

Reply:
Indeed! v8-compile-cache originally was a runtime dep, but it proved impractical so I bundled it